cg/cli/add.py

```diff
@@ -278,6 +279,8 @@ def add_case(
         priority=priority,
         ticket=ticket,
     )
+    order: Order = status_db.get_order_by_ticket_id(int(ticket))
+    new_case.orders.append(order)
```

Comment:
Do we want support for creating an order if it is not existing already? Or is it better to get an error if that happens?

Reply:
I think it is better if an error is thrown. Seems the most intuitive to me if orders should only are created when a customer places an order via the order portal

Reply:
I don't exactly know the extent of the new order flow to be honest. If a new order is created does that trigger something else somewhere, for example invoicing etc? Or are we planning to do so?

How is this used mostly? I guess in the context of the bug it was likely some development thing, should this be considered?

Reply:
The Order table was introduced to track analyses in trailblazer which belonged to the same order/ticket. Given that we soon will switch ticketing system and that cases can belong to many tickets either way, it seemed reasonable to have a many-to-many solution. 

There are quite a lot of areas where we could use the Order entries, but as of yet, it is only used for new views in CiGRID/Trailblazer. I would like to have a project in the future where we consider other areas which would benefit from using the table though.

In short, a new order does not trigger anything, but hopefully we could find some use cases in the future.

Reply:
Regarding test cases, it is a good point. Currently there is no available option to create an order via the CLI, so the tester would need to either connect to the database or submit an orderform in stage. Maybe that speaks for the option that a missing order should be created? Maybe with a click.confirm and warning message?

Reply:
Yes, I think so. 

> Regarding test cases, it is a good point. Currently there is no available option to create an order via the CLI, so the tester would need to either connect to the database or submit an orderform in stage. Maybe that speaks for the option that a missing order should be created? Maybe with a click.confirm and warning message?

Yes, I think so. But if orders will trigger something in future this would require all users to be aware of this and knowledge on other parts of the system. We likely need to document this somehow then.

Reply:
I guess? I mean it is slightly weird already that you can set a non-existent ticket on your case. Currently it is just a database model intended to contain data related to an order/ticket. Oh well, for a future discussion!